app/views/payment/student-license/PurchaseView.vue

```diff
@@ -233,6 +315,12 @@ export default {
     getSelectedPrice () {
       return this.priceData.find((p) => p.id === this.selectedPrice)
     },
+    async sendVerificationLink () {
+      if (!this.isTecmilenioEmailValid()) {
+        return false
+      }
+      await fetchJson('/db/tecmilenio/payment-link', { json: { email: this.studentEmail }, method: 'POST' })
```

Comment:
let's show a successMsg: `Por favor revise su correo electrónico para obtener el enlace de pago para completar el proceso de pago.` if API returns ok in #0B6125 color like error message so that the user knows they should check their email